README.md

```diff
@@ -39,7 +39,7 @@ To run a Docker container with [Work Directory](https://github.com/jenkinsci/rem
 Optional environment variables:
 
 * `JENKINS_JAVA_BIN`: Path to Java executable to use instead of the default in PATH or obtained from JAVA_HOME
-* `JENKINS_JAVA_OPTS` : Java Options to use for the remoting process, otherwise obtained from JAVA_OPTS
+* `JENKINS_JAVA_OPTS` : Java Options to use for the remoting process, otherwise obtained from JAVA_OPTS, **Warning!** :exclamation: mandatory double quotes around this value for windows in case of specials characters, for example `-XX:+PrintCommandLineFlags` need to be surrounded by quotes.
```

Comment:
@slide what do you think of this wording ?

Reply:
It's not the whole value though, only the items inside the value that need the quoting. You don't want the whole string quoted only the items in the string that need it.

Reply:
not sure to be able to correctly words it, can I asked our english native speaker friends to help us out here ? (@MarkEWaite @Kmartens)

Reply:
Maybe we can reference this page? https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_parsing?view=powershell-7.3 I am not sure an inline comment will be able to say what needs to be said. Can we add a link to another portion of the README that has a more detailed set of instructions? Something like 

```
* `JENKINS_JAVA_OPTS` : Java Options to use for the remoting process, otherwise obtained from JAVA_OPTS, **Warning!** :exclamation: For more information on Windows usage, please see the [section below](#windows-java-opts).
```

And then provide a new section "Windows Jenkins Java Opts" with something like this:

```
The processing of the JENKINS_JAVA_OPTS environment variable or -JenkinsJavaOpts command line parameter follow the [https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_parsing?view=powershell-7.3](command parsing semantics of Powershell). This means that if a parameter contains any characters that are part of an expression in Powershell, it will need to be surrounded by quotes. For example:

-XX:+PrintCommandLineFlags --show-version

This would need to be escaped with quotes like this:

"-XX:+PrintCommandLineFlags" --show-version

Or another example:
-Dsome.property=some value --show-version

This would need to be escaped like this:

"-Dsome.property='some value'" --show-version

```

The formatting would need to be fixed up and the wording possibly as well.